ci/e2e/terraform_provider_ocm_files/main.tf

```diff
@@ -33,6 +33,8 @@ data "aws_caller_identity" "current" {
 
 resource "ocm_cluster_rosa_classic" "rosa_sts_cluster" {
   name               = var.cluster_name
+  version            = "openshift-v${var.openshift_version}"
```

Comment:
reading the code, you don't need ``"openshift-v${var.openshift_version}"``, just ``var.openshift_version``
see: https://github.com/terraform-redhat/terraform-provider-ocm/blob/1bc6fa6f4259897967199a497488a2194e641a46/provider/cluster_rosa_classic_resource.go#L705

Reply:
ok.  hope it stays...